lms/djangoapps/verify_student/api.py

```diff
@@ -33,3 +34,42 @@ def send_approval_email(attempt):
     else:
         email_context = {'user': attempt.user, 'expiration_datetime': expiration_datetime.strftime("%m/%d/%Y")}
         send_verification_approved_email(context=email_context)
+
+
+def create_verification_attempt(user, name, status, expiration_datetime=None):
+    """
+    Create a verification attempt.
```

Comment:
I think we should document that this method is intended to be used by IDV implementation plugins to create `VerificationAttempts` - same for `update_attempt_status`.